libsrc/apple2/ser/a2.gs.s

```diff
@@ -127,7 +138,7 @@ BaudTable:                              ; bit7 = 1 means setting is invalid
                 .byte   $05             ; SER_BAUD_19200
                 .byte   $06             ; SER_BAUD_38400
                 .byte   $07             ; SER_BAUD_57600
-                .byte   $FF             ; SER_BAUD_115200
+                .byte   $00             ; SER_BAUD_115200
```

Comment:
It does not in this way - the baud rate generator's time constants are not even loaded in 115200 mode. Instead, another register instructs the chip to use the /RTxC and /SYNC pins to drive the clock. 
Maybe that warrants a comment at this place?

Reply:
Looking at the bytes spent for getClockSource, I wondering if it might result in less code if here `$40` would be used and down below...
```
        tay
        lda     BaudTable,y             ; Get chip value from Low/High tables
        bpl     BaudOK                  ; Verify baudrate is supported
```
...would be replaced with...
```
        tax
        bit     BaudTable,x             ; Get chip value from Low/High tables
        bpl     BaudOK                  ; Verify baudrate is supported
        bvc/bvs ????
```
We can use the 65C02 specific `bit <abs>,x` as we know we're on a IIgs.

Reply:
I went to bed and will check tomorrow, but I think X is held up by the channel

Reply:
Yep, X is held up with the channel value. I've tried various experiments (clock source in bit 6 of BaudTable, or an addition to BaudLow/BaudHighTable with clock source as high bit), the best I could find was -4 bytes at the price of convoluted, hard to understand code.

Reply:
I can save 5 bytes using the fact that SER_PARAMS::BAUDRATE = 0 though :)